### PR TITLE
Design edits

### PR DESCRIPTION
**Pull request by sammurraytuesta**
Update entire design, add servlets, and fetch onto page

### CODE REVIEW

**Comment by rjkroege** [comment]
This PR should be only the server side functionality right? So it shouldn't contain the README updates etc. Each PR (including *all* of its commits) should be "one" well-bounded feature. The subsequent commits to that PR should only be to resolve review feedback. A new feature should be a new PR and not include as-yet un-merged code found in other PRs. Remember to start new PRs by branching from the *main* branch.

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -1,19 +1,216 @@
...
+                <li><a href="#about" class="scroll">About</a></li>
+                <li><a href="#projects" class="scroll">Projects</a></li>
+                <li><a href="#social" class="scroll">Social</a></li>
+            </ul>
+        </nav>
     </div>
-  </body>
+
+    <section id="home">
+        <svg class="home-blob" viewBox="0 0 200 200" xmlns="http://www.w3.org/2000/svg">
```
Comment (rjkroege) [inline portfolio/src/main/webapp/index.html:28]: In general, I'd recommend having SVGs as external files because they tend not to be edited by hand.

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -1,19 +1,216 @@
...
+
+            <div class="signature">Samantha</div>
+        </div>
+
+    </section>
+
+    <footer>&copy;2020 Samantha Murray Tuesta</footer>
+
+    <!-- smooth scroll -->
+    <script src="https://code.jquery.com/jquery-3.4.1.min.js"></script>
```
Comment (rjkroege) [inline portfolio/src/main/webapp/index.html]: Be sure to read http://youmightnotneedjquery.com. Like the site says... *you might not need jquery*.   :-)

File: portfolio/src/main/webapp/js/smoothscroll.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+        $('html,body').animate({
+            scrollTop: $(this.hash).offset().top
+        }, 2000)
+    });
+
+    /* scrollreveal -  */
+
+    //scrollReveal().reveal('.showcase-text');
+
+   /*  sr.reveal('.showcase-text',{
```
Comment (rjkroege) [inline portfolio/src/main/webapp/js/smoothscroll.js]: Remove commented out code.

File: portfolio/src/main/webapp/js/smoothscroll.js [hunk 4]
```diff
@@ -0,0 +1,40 @@
+$('document').ready(()=>{
```
Comment (rjkroege) [inline portfolio/src/main/webapp/js/smoothscroll.js]: There are significant negative performance consequences to how you have implemented scrolling. You should consider the pros and cons carefully. In particular:

* using script to mediate scroll means that it won't work until the js is loaded and parsed.
* What happens if the user has JavaScript turned off?
* What happens if the user has a slow Internet connection and the script isn't loaded yet?

File: walkthroughs/week-1-web-development/examples/stanley/pom.xml [hunk 5]
```diff
@@ -12,7 +12,7 @@
     <maven.compiler.source>11</maven.compiler.source>
     <maven.compiler.target>11</maven.compiler.target>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <jetty.version>11.0.0</jetty.version>
+    <jetty.version>9.4.31.v20200723</jetty.version>
```
Comment (rjkroege) [inline walkthroughs/week-1-web-development/examples/stanley/pom.xml:15]: The pom changes probably shouldn't be part of this change?

**Review by rjkroege — CHANGES_REQUESTED** [review]

File: walkthroughs/week-1-web-development/examples/stanley/pom.xml (hunk 5 above)
Comment (rjkroege) [reply walkthroughs/week-1-web-development/examples/stanley/pom.xml:15]: You should have argued about this per https://sites.google.com/sps-program.com/students/technical-resources/jetty-9

File: portfolio/src/main/webapp/js/smoothscroll.js (hunk 4 above)
Comment (sammurraytuesta) [reply portfolio/src/main/webapp/js/smoothscroll.js]: The use of the anchor tag with the href set to the position on the site should provide an alternative to this, right? I tested this with javascript turned off and the scrolling is then just ignored and you are navigated to the section on the page like normal. What would your recommended approach be?

File: portfolio/src/main/webapp/js/smoothscroll.js (hunk 3 above)
Comment (sammurraytuesta) [reply portfolio/src/main/webapp/js/smoothscroll.js]: Thank you!

File: portfolio/src/main/webapp/index.html (hunk 2 above)
Comment (sammurraytuesta) [reply portfolio/src/main/webapp/index.html]: Haha you're right! I didn't know there was a css trick for smooth scrolling! Thank you!

File: portfolio/src/main/webapp/js/smoothscroll.js (hunk 4 above)
Comment (sammurraytuesta) [reply portfolio/src/main/webapp/js/smoothscroll.js]: I found a way using css that enables a smooth scroll that now renders this moot haha! Thank you!

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (sammurraytuesta) [reply portfolio/src/main/webapp/index.html:28]: Gotcha! In this case, since I am manipulating the css to have a fill of an image or even of a variable from the root as the set color for fill or stroke I think the use of html allows for easier css application! Do you have a recommended approach for the application of css or would it be alright to continue using the method that I have implemented?
